src/App.js

```diff
@@ -1,109 +1,31 @@
-import { useState } from 'react';
-
+import { useContext } from 'react';
+import CardCtx from './Helpers/card-context.js';
```

Comment:
Well, context is not a helper but a native way to manage state.... Putting it to a separate 'context' folder woul be great.